docs/book/src/user/personas.md

```diff
@@ -0,0 +1,178 @@
+# Personas
+
+This document describes the personas for the Cluster API 1.0 project as driven
+from use cases.
+
+We are marking a “proposed priority for project at this time” per use case.
+This is not intended to say that these use cases aren’t awesome or important.
+They are intended to indicate where we, as a project, have received a great deal
+of interest, and as a result where we think we should invest right now to get
+the most users for our project. If interest grows in other areas, they will
+be elevated. And, since this is an open source project, if you want to drive
+feature development for a less-prioritized persona, we absolutely encourage
+you to join us and do that.
+
+## Use-case driven personas
+
+### Service Provider: Managed Kubernetes
+
+Managed Kubernetes is an offering in which a provider is automating the
+lifecycle management of Kubernetes clusters, including full control planes
+that are available to, and used directly by, the customer.
+
+Proposed priority for project at this time: High
+
+There are several projects from several companies that are building out
+proposed managed Kubernetes offerings (Project Pacific’s Kubernetes Service
+from VMware, Microsoft Azure, Google Cloud, Red Hat) and they have all
+expressed a desire to use Cluster API. This looks like a good place to make
+sure Cluster API works well, and then expand to other use cases.  
+
+**Feature matrix**
+
+|   |   |
```

Comment:
I think you can eliminate this row so it doesn't show up as a blank top row in the table? (ditto for below)

Reply:
i had tried removing the header line initially, but i think the test tooling i am using is not detecting the table. i downloaded the mdbook rust application, it looked like that was the proper tool, and when i take out those lines i see this:

![image](https://user-images.githubusercontent.com/190649/73891753-eaa5d780-4842-11ea-8af3-7e5415608116.png)

Reply:
Only html `<table>` markup will allow you to get rid of the header row.

Some stackoverflow comments suggests `| <span> |` will shrink the header row to half the size, but otherwise to eliminate it. ymmv

Reply:
i'm happy to change this to a proper html table, if people don't mind the verbosity.

Reply:
Let's leave it as is. Thanks!